### PR TITLE
backport of HCAL iterative phi symmetry calibration AlCaReco for 74X

### PR DESCRIPTION
**Pull request by nilina**
Backport of 75X PR #9075

### CODE REVIEW

File: Configuration/EventContent/python/AlCaRecoOutput_cff.py [hunk 1]
```diff
@@ -38,6 +38,9 @@
 ###############################################################
 # ECAL calibration with isol. electrons
 from Calibration.EcalAlCaRecoProducers.ALCARECOEcalCalIsolElectron_Output_cff import *
+from Calibration.EcalAlCaRecoProducers.ALCARECOEcalUncalIsolElectron_Output_cff import *
+from Calibration.EcalAlCaRecoProducers.ALCARECOEcalRecalIsolElectron_Output_cff import *
+
```
Comment (mmusich) [inline Configuration/EventContent/python/AlCaRecoOutput_cff.py]: Hi @nilina why do these lines (unrelated to your PR appear here)? Have you started from the last 74X IB?

File: Configuration/StandardSequences/python/AlCaRecoStreams_cff.py [hunk 2]
```diff
@@ -46,7 +46,13 @@
...
 ###############################################################
 # ECAL calibration with isol. electrons
+# -- alcareco
 from Calibration.EcalAlCaRecoProducers.ALCARECOEcalCalIsolElectron_cff import *
+# -- alcaraw 
+from Calibration.EcalAlCaRecoProducers.ALCARECOEcalUncalIsolElectron_cff import *
+# -- alcarereco (rereco done starting from alcaraw
+#from Calibration.EcalAlCaRecoProducers.ALCARECOEcalRecalIsolElectron_cff import *
+
 ###############################################################
```
Comment (mmusich) [inline Configuration/StandardSequences/python/AlCaRecoStreams_cff.py]: also these lines look suspicious

File: Configuration/EventContent/python/AlCaRecoOutput_cff.py (hunk 1 above)
Comment (nilina) [inline Configuration/EventContent/python/AlCaRecoOutput_cff.py]: Yes, started from the last 74X. But it seems for me now that I mixed it in a hurry... I'll try to fix it!

**Comment by nilina** [comment]
Due to mixing files I created a new PR#9417 and I'm closing this one

**Pull request closed without merging**